### PR TITLE
Fixed GetContextName, updated go version

### PR DESCRIPTION
**Pull request by dwertent**
* Fixed `GetContextName` so it will return cached context name
* Updated the go version to `1.19`

### CODE REVIEW

File: k8sinterface/k8sconfig.go [hunk 1]
```diff
@@ -98,13 +97,18 @@ func GetK8sConfig() *restclient.Config {
 }
 
 // DEPRECATED
```
Comment (amirmalka) [inline k8sinterface/k8sconfig.go:99]: Why are we updating this function if it is deprecated?

File: k8sinterface/k8sconfig.go [hunk 2]
```diff
@@ -98,13 +97,18 @@ func GetK8sConfig() *restclient.Config {
...
+func GetCurrentContext() *clientcmdapi.Context {
+	if kubeConfig := GetConfig(); kubeConfig != nil {
+		kubeConfig.CurrentContext = clusterContextName
+		if clusterContextName != "" {
+			if c, ok := kubeConfig.Contexts[clusterContextName]; ok {
+				return c
+			}
+		}
+		// if context name is not set, return the current context
+		return kubeConfig.Contexts[kubeConfig.CurrentContext]
```
Comment (amirmalka) [inline k8sinterface/k8sconfig.go:109]: `kubeConfig.CurrentContext` will always be `clusterContextName` (due to assignment in line 102).. or am I missing something?

Comment (dwertent) [reply k8sinterface/k8sconfig.go:108]: You are right

File: k8sinterface/k8sconfig.go (hunk 1 above)
Comment (dwertent) [reply k8sinterface/k8sconfig.go:99]: I dont know who is using this function. It will probably be deleted soon.

**Review by amirmalka — APPROVED** [review]

**Pull request merged**